blog/_posts/2017-06-30-collections-0.2.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: Redesigned collections v0.2.0
+---
+
+We are happy to announce the
+[0.2.0 release](https://index.scala-lang.org/scala/collection-strawman/collection-strawman/0.2.0)
+of the redesigned collections!
+
+You can read more about our effort to redesign the standard collections in this previous
+[blog post](http://www.scala-lang.org/blog/2017/02/28/collections-rework.html).
+
+## Current status
+
+The goal of this release is to provide a drop-in replacement of the current collections
```

Comment:
Isn't that the general goal of the entire collections project? Not this release specifically...